cent/exceptions.py

```diff
@@ -0,0 +1,66 @@
+from cent.methods.base import CentMethod, CentType
+
+
+class CentError(Exception):
+    """
+    Wrapper for all exceptions coming from this library.
+    """
+
+
+class CentNetworkError(CentError):
```

Comment:
Let's be consistent - `Cent` prefix is used in some exaceptions, but not used in others